packages/sysfsutils.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Sysfsutils < Package
+  description 'Libsysfs provides a stable programming interface to sysfs and eases querying system devices and their attributes.'
+  homepage 'https://github.com/Distrotech/sysfsutils'
+  version '2.1.0'
+  source_url 'http://http.debian.net/debian/pool/main/s/sysfsutils/sysfsutils_2.1.0+repack.orig.tar.gz'
```

Comment:
Why don't you used the source from the github repo?

Reply:
The github repo doesn't have releases. Debian just packed the latest git source code into a tarball ( From 2010 )

Reply:
How about a compromise? I got the source code from SourceForge.